src/mono/wasi/build/WasiSdk.Defaults.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <WASI_SDK_PATH Condition="'$(WASI_SDK_PATH)' == '' or !Exists('$(WASI_SDK_PATH)/VERSION24')">$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), '..', 'wasi-sdk'))</WASI_SDK_PATH>
+    <WASI_SDK_PATH Condition="'$(WASI_SDK_PATH)' == '' or !Exists('$(WASI_SDK_PATH)/VERSION')">$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), '..', 'wasi-sdk'))</WASI_SDK_PATH>
```

Comment:
Should that be `WASI-SDK-VERSION-25.0` instead of `VERSION`?

Reply:
I believe this is the `VERSION` file that wasi-sdk ships with.  I don't know why we also write `WASI-SDK-VERSION-25.0` as well?

Reply:
Correct, `VERSION` is in the sdk .tar file.

We touch `WASI-SDK-VERSION-25.0` only do that inside of the runtime repo. 
We do it because it's easy way how to sanity-check version of the WASI-SDK in various places of MSBuild/CI pipeline.
Because it's one-liner `Condition`, instead of `<ReadLinesFromFile ...` in a `Target`

Does that make sense ?

Obviously it could be refactored, into `Target` in some common place. I don't know where to ...

Reply:
We should do something better for user machines.
There isn't a shared MSBuild file for runtime & user machine build.

Reply:
The users machine check is the actual `ReadLinesFromFile`, no need to create dummy file there.

Reply:
If you mean better than this, let's improve that in next PR.